config/locales/two_factor_authentication/en.yml

```diff
@@ -32,7 +32,7 @@ en:
     login_intro: You set these up when you created your account.
     login_options:
       auth_app: Authentication app
-      auth_app_info_html: Use your authentication application to get security code.
+      auth_app_info_html: Use your authentication application to get a security code.
```

Comment:
👀 unnecessary `_html` in the key name?

Reply:
> 👀 unnecessary `_html` in the key name?

There's quite a few of them here by the looks of it. I might do a quick a quick follow-up pull request to address them all.

Reply:
I think previously these strings were referenced with interpolated variable, hence the blanket usage of `_html` suffix.

Related: #5706